### PR TITLE
Xh augmentation

### PR DESCRIPTION
**Pull request by Ryanseanlee**

### CODE REVIEW

File: client-app/src/desktop/common/grid/SampleGridModel.ts [hunk 1]
```diff
@@ -117,7 +117,7 @@ export class SampleGridModel extends HoistModel {
                 return a < b ? -1 : 1;
             }
         },
-        restoreDefaultsFn: () => this.restoreDefaultsFn(),
+        restoreDefaultsFn: async () => this.restoreDefaultsFn(),
```
Comment (lbwexler) [inline client-app/src/desktop/common/grid/SampleGridModel.ts:120]: is this needed?

**Review by lbwexler — APPROVED** [review]
a couple of unrelated changes, but looks good!

File: client-app/src/desktop/AppModel.ts [hunk 2]
```diff
@@ -22,22 +22,13 @@ import {panelsTab} from './tabs/panels/PanelsTab';
 import {fmtDateTimeSec} from '@xh/hoist/format';
 import {span} from '@xh/hoist/cmp/layout';
 
-
 declare module '@xh/hoist/core' {
 // eslint-disable-next-line
     interface HoistUser {
```
Comment (lbwexler) [inline client-app/src/desktop/AppModel.ts:27]: we should go ahead move this to the bootstrap file as well

File: client-app/src/examples/recalls/RecallsPanelModel.ts [hunk 3]
```diff
@@ -35,8 +35,8 @@ export class RecallsPanelModel extends HoistModel {
                 {name: 'recallDate', type: 'localDate'},
                 {name: 'description', type: 'string'},
                 {name: 'reason', type: 'string'}
-            ]
-        },
+            ]}
```
Comment (lbwexler) [inline client-app/src/examples/recalls/RecallsPanelModel.ts:38]: this formatting change unrelated and likely a mistake

File: client-app/src/desktop/common/grid/SampleGridModel.ts (hunk 1 above)
Comment (Ryanseanlee) [reply client-app/src/desktop/common/grid/SampleGridModel.ts:120]: Not needed! Think this came from before our change last week

**Pull request merged**